cloud/api-gateway/src/main/java/com/qpay/apigateway/exception/InvalidTokenException.java

```diff
@@ -4,8 +4,8 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 
 @ResponseStatus(code = HttpStatus.UNAUTHORIZED)
-public class MissingTokenException extends RuntimeException{
-    public MissingTokenException() {
-        super("Unauthorized access");
+public class InvalidTokenException extends RuntimeException{
```

Comment:
it's not the task of api-gateway decide type of errors. The only task is to return error from auth-manager